collectors/30/yarn_mapreduce.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+import time
+
+from collectors.lib.optimizely_utils import format_tsd_key, get_json
+
+
+# Constants
+TIME = int(time.time())
+HOST_URL = 'http://127.0.0.1:8088'
+HOST_URL = 'http://hbaseprod1-1b-east-0024b866-exclusiverat:8088'
```

Comment:
Did you mean to leave in the remote url?

Reply:
Nope, fixed-up.